Makefile

```diff
@@ -372,10 +372,7 @@ GIT_REV:=$(shell git rev-parse --short HEAD)
 deploy-olm: THIS_OPERATOR_IMAGE?=ttl.sh/oadp-operator-$(GIT_REV):1h # Set target specific variable
 deploy-olm: THIS_BUNDLE_IMAGE?=ttl.sh/oadp-operator-bundle-$(GIT_REV):1h # Set target specific variable
 deploy-olm: DEPLOY_TMP:=$(shell mktemp -d)/ # Set target specific variable
-deploy-olm: operator-sdk ## Build current branch operator image, bundle image, push and install via OLM
-	oc whoami # Check if logged in
-	oc create namespace $(OADP_TEST_NAMESPACE) || true
-	$(OPERATOR_SDK) cleanup oadp-operator --namespace $(OADP_TEST_NAMESPACE)
+deploy-olm: operator-sdk undeploy-olm ## Build current branch operator image, bundle image, push and install via OLM
```

Comment:
I am not good with makefile, did I do what I wanted? (run `undeploy-olm` prior to `deploy-olm`)